Versions/Common/logic.class.lua

```diff
@@ -287,7 +287,7 @@ end
 -- Check for Flight Form
 -- Returns: boolean
 function BeStride:DruidCanFly()
-	if IsUsableSpell(BeStride_Constants.spells.druid.flightform) then
+	if IsUsableSpell(BeStride_Constants.spells.druid.flightform) or (BeStride_Game == "Wrath" and GetShapeshiftForm() == 6) then
```

Comment:
`IsUsableSpell()` returns false if the druid is in flying form and flies on Wrath, I guess this is not the case on Mainline, and I had to fix that to let the player dismiss flying form when flying.  I'm not sure I did the right thing.

Reply:
Try IsPlayerSpell() instead

Reply:
Done, using `IsPlayerSpell()` instead of `IsUsableSpell()`. That works as expected on Wrath.